README.md

```diff
@@ -305,11 +317,17 @@ ruff format .
 If you would prefer to see what would change you can:
 
 ```shell
-ruff format --check .
+ruff format --check . # just tells you that things would change, e.g. "1 file would be reformatted, 4 files already formatted"
+ruff format --diff .  # if files would be reformatted: print the diff between the current file and the re-formatted file, then exit with non-zero status code
 ```
 
 Similarly if you would like to see if there are any type checking errors you can:
 
 ```shell
 mypy .
 ```
+
+One line for running the linter, the type checker, and the test suite (failing fast if there are errors):
+```shell
+ruff format --diff . && ruff check && mypy . && pytest
```

Comment:
i use this all the time when i'm developing, nice to have a thing to just copy/paste